### ISSUE TITLE
LDAP-Based URL for Oracle Database

### ISSUE THREAD

**Issue by gilliM** [sfa-siard/siard-suite#97]
We have an Oracle database for which we need to use a LDAP-based url for authentication
https://docs.oracle.com/en/applications/enterprise-performance-management/11.2/hitju/ldap_based_url_for_oracle_database.html

It seems that it's currently not possible to use it with SIARD-suite. The interactive updates between the interface fields and the check (semi-column in host adress) are maybe blocking here.

**Comment by mburri** [comment]
@gilliM 
Yeah, that wont't work for now. You could try to use the command line tool (see the documentation on how to use it)

But there might be some validation in place that will reject jdbc urls with ldap based auth.

**Comment by gilliM** [comment]
@mburri thanks for the answer. I kind of managed to make it work with the command line tool. When trying to target our production database, i get a

>  java.sql.SQLRecoverableException: I/O-Fehler: Undefined Error< oracle.net.ns.NetException: Undefined Error

Is there a way to see a log or a verbose mode ?

**Comment by mburri** [comment]
There should be a file 'siardcmd.log' in your users home folder. 
There might be more info available there.
I have to check how  to change the log level if needed.
